torchgeo/datasets/cowc.py

```diff
@@ -101,7 +101,9 @@ def __init__(
         self.images = []
         self.targets = []
         with open(
-            os.path.join(self.root, self.filename.format(split)), newline=""
+            os.path.join(self.root, self.filename.format(split)),
+            encoding="utf-8-sig",
```

Comment:
The files we test on were ASCII, but the actual downloaded files were UTF-8 Unicode (with BOM) text. This would result in some extra header garbage being added to the name of the first image filepath. I updated the test files to be UTF-8 Unicode (with BOM) text so we can properly test this.

Reply:
good catch!